board/piksiv3/rootfs-overlay/etc/resolv.conf

```diff
@@ -0,0 +1,2 @@
+nameserver 8.8.8.8
+nameserver 8.8.4.4
```

Comment:
Is there a better way to get a DNS configuration in? Without this, we've got no resolving capability - see https://github.com/swift-nav/piksi_buildroot/issues/147. @gsmcmullin what's the better way to handle this? No, hasn't been tested with DHCP.